CHI_TECH/ChiPhysics/PhysicsMaterial/transportxsections/prop_transportxs_03_ecollapse.cc

```diff
@@ -77,37 +78,38 @@ void chi_physics::TransportCrossSections::
   //initial guess of 1.0. Here we reset them
   for (int g=0; g<G; g++)
     if (sigma_tg[g] < 1.0e-16)
-      A(g,g) = 1.0;
+      A[g][g] = 1.0;
+
+  MatDbl Ainv = math.Inverse(A);
+  MatDbl C    = math.MatMul(Ainv,B);
+  VecDbl E(G,1.0);
+  VecDbl E_new(G,0.0);
 
-  Eigen::MatrixXd Ainv = A.inverse();
-  Eigen::MatrixXd C = Ainv*B;
-  Eigen::VectorXd E(G); E.setConstant(1.0);
-  Eigen::VectorXd E_new(G);
 
   //============================================= Perform power iteration
```

Comment:
Could we use here the power iteration routine from ChiMath?

Reply:
Done. Good catch actually I thought this was already done.